instance/models/mixins/openedx_database.py

```diff
@@ -235,6 +235,10 @@ def _get_mysql_settings(self):
 
             # ecommerce
             "ECOMMERCE_DEFAULT_DB_NAME": self._get_mysql_database_name("ecommerce"),
```

Comment:
Could you please add a comment here explaining that this variable has been replaced by `ECOMMERCE_DATABASE_NAME`, but that we're keeping this one around to maintain compatibility with Ginkgo?

Reply:
I've moved all old style settings together and put a comment explaining why they are there.